app/routes.js

```diff
@@ -30,4 +30,9 @@ exports.init = app => {
     [authenticationMiddleware.validate, authorizationMiddleware.ownerAdmin],
     usersController.listAlbums
   );
+  app.post(
+    '/users/sessions/invalidate_all',
```

Comment:
The `all` is not necesary. Its already implied by rest, because to invalidate a single session the route should be something like `/users/sessions/{:session_id}/invalidate`